betelgeuse.py

```diff
@@ -97,6 +98,17 @@ def parse_junit(path):
     return result
 
 
+def parse_test_results(test_results):
+    """Returns the summary of test results by their status.
+
+
+    :param str path: Path to the jUnit XML file.
+    :return: A list of dicts with information about every test
+        case result.
+    """
+    return Counter([test['status'] for test in test_results])
```

Comment:
@elyezer I chose to create this "standalone" function because it be can called from other scripts, which is what I want to do with report generating scripts I intent to work on for Jenkins.